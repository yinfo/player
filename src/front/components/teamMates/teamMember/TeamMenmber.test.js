// @flow

import React from 'react';
import { shallow } from 'enzyme';
import TeamMember from './TeamMember';

describe('TeamMember component', () => {
  it('renders as expected', () => {
    const props = {
      picture: '',
      firstname: '',
      lastname: '',
      profile: '',
      profileColor: 'info',
    };
    const component = shallow(<TeamMember {...props} />);
    expect(component).toMatchSnapshot();
  });
});
