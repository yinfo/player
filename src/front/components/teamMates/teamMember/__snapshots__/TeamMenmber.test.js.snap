// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamMember component renders as expected 1`] = `ShallowWrapper {}`;
